docs/api/13_wallets/wallet-object.mdx

```diff
@@ -0,0 +1,79 @@
+---
+sidebar_position: 1
+---
+
+# Wallet object
+
+This object represents a wallet.<br></br>
+
+## Schema
+
+```json
+{
+  "wallet": {
+    "lago_id": "b7ab2926-1de8-4428-9bcd-779314ac129b",
+    "lago_customer_id": "12345",
+    "external_customer_id": "12345678",
+    "status": "active",
+    "currency": "EUR",
+    "name": "name",
+    "rate_amount": 1.5,
+    "credits_balance": 10,
+    "balance": 10,
+    "consumed_credits": 2,
+    "created_at": "2022-04-29T08:59:51Z",
+    "expiration_date": null,
+    "last_balance_sync_at": "2022-04-29T08:59:51Z",
+    "last_consumed_credit_at": "2022-04-29T08:59:51Z"
+  }
+}
+```
+
+| Attributes | Description |
+| -----------| ----------- |
+| **lago_id** &nbsp &nbsp <Type>String</Type> &nbsp &nbsp <NotNullable>Not null</NotNullable> | Unique identifier of the wallet in Lago application. |
+| **lago_customer_id** &nbsp &nbsp <Type>String</Type> &nbsp &nbsp <NotNullable>Not null</NotNullable> | Internal Lago customer ID. |
+| **external_customer_id** &nbsp &nbsp <Type>String</Type> &nbsp &nbsp <NotNullable>Not null</NotNullable> | External Lago customer ID. |
```

Comment:
"External customer ID" (would remove "Lago" to avoid confusion)

Reply:
fixed